draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -473,6 +473,17 @@ The first four bits are used to specify compatible versions of the
 MP-DCCP implementation. The following four bits are reserved for further
 use.
 
```

Comment:
Ok, and I'd also suggest to improve the description because it's not really clear how those 4-bits are supposed to indicate the version(s): is it a bitfield which contains a list of supported versions (i.e. bit #0->v0, bit#1->v1, etc..) or is it just a 4-bit value representing the highest supported version number like in MPTCP?

Reply:
Or are we supposed to follow the same "reconciliation rule" described in 6.3.1 of RFC4340, i.e. use the 4-bit field as version number and repeat it for each supported version in order of preferred priority?

Reply:
1. 4-bit value like in MPTCP
2. repeated for each supported version